scripts/docker-build.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+pushd `dirname $0` > /dev/null
+ST2DOCSROOT="${PWD%/*}"
+popd > /dev/null
+
+docker build -t st2/st2docs -f $ST2DOCSROOT/Dockerfile $ST2DOCSROOT
```

Comment:
More obvious thing looks like to include the instructions in a `Makefile`, so the commands would be: 

```sh
make docker-build
make docker-run
```